include/zephyr/xen/regions.h

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright (c) 2023 EPAM Systems
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+#ifndef __XEN_REGIONS_H__
+#define __XEN_REGIONS_H__
+
+/*
```

Comment:
I think Doxygen style will be better `/*` -> `/**`

Reply:
fixed